src/arduino.cc/arduino-builder/main.go

```diff
@@ -134,7 +134,15 @@ var loggerFlag *string
 var versionFlag *bool
 var vidPidFlag *string
 
+var Usage = func(){
+    fmt.Fprintf(os.Stderr, "Usage of %s:\n", os.Args[0])
+    fmt.Fprintf(os.Stderr, "\n\tarduino_builder -hardware <hardware_dir> -tools <toosl_dir> -fqbn <family:arch:model> [options] sketch_to_Compile\n\n")
+    fmt.Fprintf(os.Stderr, "Available flags:\n")
+    flag.PrintDefaults()
+}
+
 func init() {
+    flag.Usage = Usage
```

Comment:
This is needed to substitute the default flag.Usage() behaviour with a custom one (defined above).